src/main/target/FURYF4/target.h

```diff
@@ -20,6 +20,8 @@
 
 #pragma once
 
+#define USE_LATE_TASK_STATISTICS
```

Comment:
Why is this enabled here?

Reply:
Because I was debugging it on that target. Only the unified targets were at one point struggling with space for this function, so I debugged on some specific ones. I'll remove all those; FURYF4, REVO and REVONANO. See bc62841.